### ISSUE TITLE
StoreScene causes OpenGL crashes when accessed

### ISSUE THREAD

**Issue by Tanner** [Tanner/Oregon-Trail#45]
I currently cannot access StoreScene. All other scenes work. Here is the stack trace:

```
Fri Oct 21 08:06:12 EDT 2011 ERROR:Invalid value (1281)
org.lwjgl.opengl.OpenGLException: Invalid value (1281)
    at org.lwjgl.opengl.Util.checkGLError(Util.java:59)
    at org.lwjgl.opengl.GL11.glScissor(GL11.java:2537)
    at org.newdawn.slick.opengl.renderer.ImmediateModeOGLRenderer.glScissor(ImmediateModeOGLRenderer.java:266)
    at org.newdawn.slick.Graphics.setClip(Graphics.java:801)
    at org.newdawn.slick.Graphics.setClip(Graphics.java:820)
    at component.Component.render(Component.java:68)
    at component.Panel.render(Panel.java:75)
    at component.Component.render(Component.java:132)
    at scene.SceneLayer.render(SceneLayer.java:18)
    at scene.Scene.render(Scene.java:43)
    at org.newdawn.slick.state.StateBasedGame.render(StateBasedGame.java:199)
    at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:681)
    at org.newdawn.slick.AppGameContainer.gameLoop(AppGameContainer.java:408)
    at org.newdawn.slick.AppGameContainer.start(AppGameContainer.java:318)
    at core.GameDirector.start(GameDirector.java:70)
    at core.GameDirector.main(GameDirector.java:255)
Fri Oct 21 08:06:12 EDT 2011 ERROR:Game.render() failure - check the game code.
org.newdawn.slick.SlickException: Game.render() failure - check the game code.
    at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:684)
    at org.newdawn.slick.AppGameContainer.gameLoop(AppGameContainer.java:408)
    at org.newdawn.slick.AppGameContainer.start(AppGameContainer.java:318)
    at core.GameDirector.start(GameDirector.java:70)
    at core.GameDirector.main(GameDirector.java:255)
```

**Comment by Tanner** [comment]
Tagging @ryanashcraft - I have no idea what causes this other than clip. Passing in an invalid rectangle?

**Comment by Tanner** [comment]
As far as I know this is just isolated to me… and I have no idea why.

**Comment by Tanner** [comment]
Fixed by e1519efd.
